### PR TITLE
Create CHANGELOG.md

### PR DESCRIPTION
**Pull request by mishmosh**
~Awaiting contributors list~, but everything else is ready for review.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
+# go-filecoin changelog
+
+## go-filecoin 0.1.4
+
+We're happy to announce go-filecoin 0.1.4. This release contains a better install experience, Proof-of-Spacetime integration, more reliable message sending and networking, and many other improvements. Get pumped! 🎁
```
Comment (phritz) [inline CHANGELOG.md]: initial Proof of Spacetime integration? it is not the final construction

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Linux and MacOS binaries for go-filecoin are now available! See [Installing from binary](https://github.com/filecoin-project/go-filecoin/wiki/Getting-Started#installing-from-binary) for instructions.
+
+#### 🍱 Precompiled proofs parameters
+
+Running secure proofs requires parameter files that are several GB in size. Previously, these files were generated at install, an extremely memory-intensive process causing slow or impossible builds for many users. Now, you can download pre-generated files during install by running `paramfetch`. This step is now included in the [Installing from binary](https://github.com/filecoin-project/go-filecoin/wiki/Getting-Started#installing-from-binary) instructions.
+
+#### 🦖 Version checking
+
+go-filecoin now checks that it is running the same version (at the same commit) before connecting to a devnet. This is a temporary fix until a model for change is implemented, allowing different versions to interoperate.
```
Comment (phritz) [inline CHANGELOG.md]: while connecting to a devnet

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+### Features
+
+#### 💎 Proof-of-Spacetime Integration
+
+For each storage deal, miners now call `rust-fil-proofs` to periodically generate proofs of continued storage. With this major integration in place, you can expect some follow-up  (for example, storage mining faults do not yet appear on-chain) and continued optimizations to the underlying Proof-of-Spacetime implementation.
```
Comment (phritz) [inline CHANGELOG.md]: I would just say 'miners now call...'

Comment (phritz) [inline CHANGELOG.md]: should probably say its not the final construction

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+### Changelog
+
+A full list of [all 82 PRs in this release](https://github.com/filecoin-project/go-filecoin/pulls?utf8=%E2%9C%93&q=is%3Apr+merged%3A%3E%3D2019-02-14+-merged%3A%3E%3D2019-03-06) can be found on Github.
+
+### Contributors
+
+❤️ Huge thank you to everyone that made this release possible! By alphabetical order, here are all the humans who contributed to the release:
+
+PENDING
```
Comment (phritz) [inline CHANGELOG.md]: todo

File: CHANGELOG.md (hunk 3 above)
Comment (mishmosh) [reply CHANGELOG.md]: done

Comment (mishmosh) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 2 above)
Comment (mishmosh) [reply CHANGELOG.md]: done

File: CHANGELOG.md (hunk 1 above)
Comment (mishmosh) [reply CHANGELOG.md]: done

**Comment by mishmosh** [comment]
Committers list updated, ready for review.

**Pull request merged**